src/aiida/common/hashing.py

```diff
@@ -41,7 +41,8 @@ def get_random_string(length: int = 12) -> str:
     return ''.join(secrets.choice(alphabet) for i in range(length))
 
 
-BLAKE2B_OPTIONS = {
+# Relaxed typing needed due to https://github.com/python/mypy/issues/5382
```

Comment:
This probably has a better solution, but I am not enough of a typing wizard yet.

Reply:
Since they are all integers, couldn't we define it as `dict[str, int]`? Or will this cause a false positive in `hashlib.blake2b` on line 122?

Reply:
That's the type that mypy infers, but then it gets mad. I think it's because it cannot properly unpack it, and some of the parameters for the blake2b function are not integers.